src/Campaigns/resources/admin/components/CampaignDetailsPage/Tabs/Settings.tsx

```diff
@@ -19,11 +21,75 @@ export default () => {
         formState: {errors},
     } = useFormContext();
 
-    const [goalType, image, status, shortDescription] = watch(['goalType', 'image', 'status', 'shortDescription']);
+    const [goalType, image, status, shortDescription, enableCampaignPage] = watch([
+        'goalType',
+        'image',
+        'status',
+        'shortDescription',
+        'enableCampaignPage',
+    ]);
     const isDisabled = status === 'archived';
 
     return (
         <div className={styles.sections}>
+            <div className={styles.section}>
+                <div className={styles.leftColumn}>
+                    <div className={styles.sectionTitle}>{__('Campaign page', 'give')}</div>
+                    <div className={styles.sectionDescription}>
+                        {__(
+                            'Set up a landing page for your campaign. The default campaign page has the campaign details, the campaign form, and donor wall.',
+                            'give'
+                        )}
+                    </div>
+                </div>
+                <div className={styles.rightColumn}>
+                    <div className={styles.sectionField}>
+                        <img
+                            style={{marginTop: '1rem'}}
+                            src={campaignPage}
+                            alt={__('Enable campaign page for your campaign.', 'give')}
+                        />
+                        <div className={styles.toggle}>
+                            <ToggleControl
+                                label={__('Enable campaign page for your campaign.', 'give')}
+                                help={__('This will create a default campaign page for your campaign.', 'give')}
+                                name="enableCampaignPage"
+                                checked={enableCampaignPage}
+                                onChange={(value) => {
+                                    setValue('enableCampaignPage', value, {shouldDirty: true});
+                                }}
+                            />
+                        </div>
+                        {!enableCampaignPage && (
+                            <div className={styles.warningNotice}>
+                                <svg
```

Comment:
There is an `Icons` folder in the `Campaigns/admin/components` directory, I think it will look better to move this SVG there, just for the sake of consistency.

Reply:
Resolved in aa4aa372ae22ea203577a522dfc510a6a292c8de